.travis.yml

```diff
@@ -36,15 +36,15 @@ matrix:
             - libasound2-dev
       before_script:
         - mkdir -p $HOME/graalvm && cd $HOME/graalvm
-        - wget https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1.0-ea+25.zip
+        - wget https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1-latest.zip
         - unzip graalvm-svm-linux-20.1.0-ea+25.zip && cd $TRAVIS_BUILD_DIR
         - export GRAALVM_HOME=$HOME/graalvm/graalvm-svm-linux-20.1.0-ea+25
```

Comment:
The unzip command and GRAALVM_HOME variable need to be updated as well with the new values.

Reply:
I just did it :)